### PR TITLE
Controller logic

### PR DESCRIPTION
**Pull request by aselbie**
This is WIP, but it doesn't break anything, and I wanted to commit since we're all working so closely on this. Don't take this as final, since I'm making lots of guesses about the API.

### CODE REVIEW

File: src/views/parts/timeline.jade [hunk 1]
```diff
@@ -1,6 +1,2 @@
 ul.timeline
-  li
-
-  li.active
-
-  li
\ No newline at end of file
+  li(ng-repeat="phase in vm.timeline track by $index", ng-class="phase")
```
Comment (vietnogi) [inline src/views/parts/timeline.jade]: need the class active?

File: src/scripts/submissions.controller.coffee [hunk 2]
```diff
@@ -5,6 +5,30 @@ SubmissionsController = ($scope, SubmissionAPIService) ->
...
+      number: 1
+      outOf: 3
+      name: 'Design Concepts'
+      status: 'open'
+    next:
+      name: 'Final Designs'
+      starts: '2015-05-05T20:53:41.467Z'
+
+  vm.ranks = [
+    { value: 1, longLabel: '1st Place', avatarUrl: null }
```
Comment (vietnogi) [inline src/scripts/submissions.controller.coffee]: how come its `longLabel` instead of just `label`?

**Comment by vietnogi** [comment]
looks like theres a decrease in coverage

**Comment by aselbie** [comment]
Addressing all of these this morning.

**Comment by aselbie** [comment]
Requires the changes in https://github.com/appirio-tech/api-schemas/pull/6

File: src/scripts/submissions.controller.coffee [hunk 3]
```diff
@@ -1,10 +1,31 @@
...
+    '2nd Place'
+    '3rd Place'
+    '4th Place'
+    '5th Place'
+    '6th Place'
+    '7th Place'
+    '8th Place'
+    '9th Place'
+    '10th Place'
+  ]
```
Comment (samsep) [inline src/scripts/submissions.controller.coffee]: Should these be based on the rank # in the submissions data?

Comment (aselbie) [inline src/scripts/submissions.controller.coffee]: Possibly. This is pretty rough without knowing what the API will look like. It's possible that these rank names will come from the service, but for now I'm assuming that the rank will be coming as a single zero-indexed number. It's simple to this out of we get more info.

**Comment by vietnogi** [comment]
are there difficulties with working on the controller without having an updated api schemas?  I figured it would be easy to hardcode the deta in the controller.

File: src/scripts/submissions.controller.coffee (hunk 3 above)
Comment (vietnogi) [inline src/scripts/submissions.controller.coffee]: You can make the API however you want it. When we meet, you can confirm it with luzheng.

**Comment by aselbie** [comment]
Alright. Sorry for the confusion. I've closes the schemas PR and added a method to transform the response with extra mock data.

**Comment by samsep** [comment]
LGTM

**Pull request merged**